articles/cloud-computing-cost-optimization-with-cloud-bursting/index.md

```diff
@@ -0,0 +1,73 @@
+
+### Benefits of cloud bursting
+
+#### Capacity boosting
+
+Cloud bursting ensures uninterruptable operation of organizations even in times of high demand. During peak periods, organizations can run at a higher capacity, sometimes nearing 100 percent on the private cloud. This situation causes a strain on the systems and may result in slow speeds or a complete crash. Cloud bursting reduces the strain on your private cloud by redirecting traffic to your public cloud, thus ensuring that systems continue to work at the recommended rate and capacity.
+
+When you need some extra space on your private cloud to, for example, run applications, you can always cloud burst to free up space. In this scenario, cloud bursting helps to transfer data from your private cloud to the public cloud for a period.
+
+#### Flexible scaling
+
+Organizations with fluctuating customer bases and those startups wanting to scale their operations rely on cloud bursting for rapid and convenient scaling. Cloud bursting offers an added convenience in that organizations do not need to predict demand usage manually. Besides, organizations can cloud burst to free up local resources and use them for other purposes.
+
+#### Business continuity
+
+Cloud bursting prevents critical applications from crashing; otherwise, operations could come to a complete standstill. Application crash prevention is very beneficial to organizations. It ensures organizations maintain their functions during high demand times.
+
+### Challenges of cloud bursting
+
+#### Managing configurations
+
+Organizations that cloud burst run the same resource types, for example, application servers, in multiple clouds. But choosing multiple cloud providers may bring some challenges in terms of different resource offerings. For instance, it may require an organization to manage and configure its applications and technology stacks across two environments.
+
+Differences may come in terms of cloud network configurations. Some cloud providers offer the concept of availability zones, while others do not. Still, some have security groups, others have Access Control Lists ([ACLs](https://geek-university.com/ccna/what-is-acl-access-control-list/#)) and [Subnets](https://en.wikipedia.org/wiki/Subnetwork#), and some have both.
+
+[Hypervisors](https://www.vmware.com/topics/glossary/content/hypervisor#), their features, and versions of hypervisors differ between cloud providers. Also, machine types and power vary from one cloud provider to another. Clouds also offer different storage types, including object, persistent, and block.
```

Comment:
What are hypervisors? Attach links only when you feel additional information is provided at that link and is out of the scope of this article. To understand this article, the reader needs to know what Hypervisors are. As mentioned earlier, if the reader doesn't get information in the article, he loses his attention. Make articles reader-friendly by providing all the information needed. You may include a keywords section as mentioned in the previous review if required.

Reply:
The same holds true with ACLs and subnets.